app/models/product.rb

```diff
@@ -0,0 +1,14 @@
+class Product < ApplicationRecord
+  belongs_to :merchant
+  has_many :order_items
+  has_many :orders, through: :order_items
+
+  validates :name, presence: true, uniqueness: true
+  validates :price, presence: true, numericality: { greater_than: 0 }
+
+  def self.spotlight
+    spotlight_for_all_products = Product.all.sample
+    return spotlight_for_all_products
```

Comment:
You don't need to assign the result to a variable here since you immediately return it:

```suggestion
    return Product.all.sample
```